packages.yml

```diff
@@ -8,8 +8,11 @@ packages:
   - package: fivetran/facebook_ads
     version: [">=0.5.0", "<0.6.0"]
 
-  - package: fivetran/google_ads
-    version: [">=0.8.0", "<0.9.0"]
+  # - package: fivetran/google_ads
+  #   version: [">=0.8.0", "<0.9.0"]
+  - git: https://github.com/fivetran/dbt_google_ads.git
```

Comment:
This should be reverted back to the original commented out `google_ads` package before merging I think?

Reply:
yep!